pants

```diff
@@ -49,11 +49,13 @@ fi
 
```

Comment:
Might be worth adding a comment here about the shellcheck annotations, I was confused a bit when I saw them in this file.

Reply:
I don't think I agree, because we use these directives in multiple other files and I wouldn't want to add the comment anywhere we use a directive.

This is very similar to comments we use elsewhere like `# isort:skip`. They're a bit confusing at first, but also call the tool's name right away and are very search engine friendly.

Reply:
Sure, I was only confused for like 15 seconds :)